packages/adapter-knex/lib/adapter-knex.js

```diff
@@ -507,6 +511,13 @@ class QueryBuilder {
 
     this._addWheres(w => this._query.andWhere(w), listAdapter, where, baseTableAlias);
 
+    // TODO: Implement configurable search fields for lists
+    const searchField = listAdapter.fieldAdaptersByPath['name'];
+    if (search !== undefined && searchField && searchField.fieldName === 'Text') {
```

Comment:
I'm not sure that `searchField.fieldName === 'Text'` will actually work? TBH I'm not sure, but we should really be checking `searchField.field instanceof Text` to make this more explicit.

Reply:
I tested it with different field types but happy to go with instanceof